aiohttp_sentry/__init__.py

```diff
@@ -1,22 +1,49 @@
+import asyncio
 from functools import partial
+import sys
 
 import raven
 import raven_aiohttp
 
 
 class SentryMiddleware:
 
-    def __init__(self, sentry_kwargs=None):
+    def __init__(self, install_excepthook=True, sentry_kwargs=None):
```

Comment:
Many are setting `sentry_kwargs` as the first positional argument, so please add `install_excepthook` as the second argument instead to not break backwards compatibility. Also, to have a nicer API, please make this flag keyword-only, like this:

```python
    def __init__(self, sentry_kwargs=None, *, install_excepthook=True):
```